### PR TITLE
CEM-888 ensured usage of FeeTypeEnum

### PR DESCRIPTION
**Pull request by DenisMela**
**Checklist:**
- [ ] Changed package version (must be higher than **master**)
- [ ] Updated README (if applicable)

### CODE REVIEW

File: src/app/controllers/MenuItemController.ts [hunk 1]
```diff
@@ -5,6 +5,10 @@ import { FlashSaleItem } from "./FlashSaleController";
 import { DefaultControllerRequired } from "./Controller";
 import { MutateResult } from "../links/synchronouslinks/GraphQLLink";
 
+export enum FeeType {
```
Comment (junthehacker) [inline src/app/controllers/MenuItemController.ts:8]: Is there any reason that you don't simply use the enum exported from enums.ts?

Comment (DenisMela) [reply src/app/controllers/MenuItemController.ts:8]: @junthehacker  The reason I didn't is I'm not sure how the graphql transpiler automatically generates the enums.ts file, I wasn't sure if what I added to Enums was to be deleted or not after it's generated again. I'll go through the PRs and fix the references, and I'll make a ticket for removing the duplicate enums in all controllers so they're all referencing from enums.ts

Comment (DenisMela) [reply src/app/controllers/MenuItemController.ts:8]: Vili explained it to me, I'll update the ts-sdk after I merge the graphql updates.

**Review by ralph-dev — APPROVED** [review]
LGTM

**Pull request merged**